tests/scanner/test_data/gce_network_rules.py

```diff
@@ -0,0 +1,8 @@
+rules:
```

Comment:
Is this the same copy of the rules file from above?  If so, can just use the copy from above, and this can be dropped.